pos_analytic_by_config/models/analytic_distribution_model.py

```diff
@@ -13,8 +13,9 @@ class AccountAnalyticDistributionModel(models.Model):
     )
 
     def _get_distribution(self, vals):
+        new_context = vals.copy()
```

Comment:
Incorrect variable name. It should be in any case `new_vals`. But why do you need to copy it?

Reply:
Because new_vals can't be updated